# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/worklet/WorkletFetchHandler.cpp
# Commit: f06186536887
# Full Hash: f061865368870d7e33c22d537992d272916ebec1
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2025-10-03 20:53 +0000
# Regressor Bug: 1968890
# File Overlap Count: 3
# Overlapping Files: dom/worklet/WorkletFetchHandler.cpp, dom/workers/ScriptLoader.cpp, dom/script/ScriptLoader.cpp
# Description:
#   Bug 1968890 - Part 6: ScriptLoaders update. r=jonco,dom-worker-reviewers,edenchuang
# ==============================================================================

diff -r 584beaae1d20 -r f06186536887 dom/worklet/WorkletFetchHandler.cpp
--- a/dom/worklet/WorkletFetchHandler.cpp	Fri Oct 03 20:53:29 2025 +0000
+++ b/dom/worklet/WorkletFetchHandler.cpp	Fri Oct 03 20:53:29 2025 +0000
@@ -220,15 +220,6 @@
 
   request->mBaseURL = mURI;
   request->OnFetchComplete(mResult);
-
-  if (NS_FAILED(mResult)) {
-    if (request->IsTopLevel()) {
-      request->LoadFailed();
-    } else {
-      request->Cancel();
-    }
-  }
-
   moduleLoader->RemoveRequest(mURI);
   return NS_OK;
 }
